nekoyume/Assets/_Scripts/Extensions/GameActionExtensions.cs

```diff
@@ -6,10 +6,12 @@ namespace Nekoyume
 {
     public static class GameActionExtensions
```

Comment:
I think it could be renamed as `ActionExtensions`

Reply:
There is the `ActionExtensions` namespace so I renamed it as `ActionBaseExtensions`.